tasks/interactive-bots.md

```diff
@@ -31,106 +31,126 @@ can be integrated with various APIs.
  Task Types below.
 
-### Task Type A: Learn about interactive bots by running the followup bot
+### Task Type A: Learn about interactive bots by running the 'followup bot'
 
-Follow the tutorial below to test run the followup bot on your local instance.
+Follow the tutorial below to test run the followup bot on your Zulip instance
+(either local or in your droplet).
 
 * Run the Zulip server and log in the browser as Cordelia.
 
 * Find the API keys.
 
-  You can get API keys just below the bots section in the settings page. This 
+  You can get API keys just below the bots section in the settings page. This
   is how Zulip knows the request is from an authorized user.
 
   You can also create a new bot for a user in settings - it will have its own
-  api key and email. Remember to subscribe it to the streams you want to use the
+  API key and email. Remember to subscribe it to the streams you want to use the
    bot in.
 
-* Create a `~/.zuliprc-local` file with credentials:
-
+* Create a `~/.zuliprc-local` file with credentials (`nano ~/.zuliprc-local`):
   ```
   [api]
-  key=<api-key>
-  email=<email>
-  site=<dev-url>
+  key=<api-key> (Of the bot)
+  email=<email> (Of the bot)
+  site=<dev-url> (localhost/Zulip server)
   ```
 
-  Replace the placeholder `<api-key>` in the example with an existing user API
-  key and placeholder `<email>` with the user email. The placeholder `<dev-url>` 
-  should point to your development environment URL. 
-  
-* Create a `followup` stream.
+* Create a `followup` stream on your Zulip server.
 
-* Use "Manage Streams" to subscribe your bot to the following streams: devel, social, and followup
+* Use `Manage Streams` to subscribe your bot (if you created one) to the following streams: `devel`,
+`social`, and `followup`.
 
-* Run the followup bot:
+* Run the followup bot outside of your VM:
 
   ```
   cd ~/zulip/contrib_bots
   python run.py lib/followup.py --config-file ~/.zuliprc-local
   ```
 
+  Make sure to point to your `.zuliprc-local` file - if you have created it in
+  a different folder than `~`, you have to point to the right folder, e.g.
+  `../../.zuliprc-local`.
+
 * Test manually that the followup bot is working on the local Zulip instance,
 by:
 
   a. sending a few messages starting with `@followup` and `@follow-up` from
-  different streams, such as "devel" and "social"
+  different streams, such as `devel` and `social`
 
   b. checking that the messages showed up in the `followup` stream, prepended by
   the sender email
 
-* Take screenshots showing that the bot is working, i.e. your terminal window
-with the bot working, messages sent by the bot in the followup stream. Add the
-screenshots to `interactive-bots/followup/<username>/`.
+* Take screenshots showing that the bot is working. Make sure to take screenshots
+of:
+
+ - the output in your terminal window with the bot running, including the command you used to
```

Comment:
this line is too long (79 chars max) please ensure that every line is shorter than 80 chars